Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/utils/mui-utils.ts

```diff
@@ -14,6 +16,17 @@ const handleDateTimePickerChange = (name: string, newValue:any, onChange: Functi
     onChange(event);
 };
 
+/**
+ * Allows search by id when added to autocomplete {filterOptions} property
+ */
+const autocompleteNameIdFormatFilterOptions = createFilterOptions({
+    stringify: (option: IHasNameAndIdType) => {
+        const { name, id } = option;
+
+        return `${name} ${id}`;
```

Comment:
Can you put '#' before the id, so it's clearer in the results view.

Reply:
This is internal formatting in the select menu. The id is not shown in the list menu text

Reply:
Done